frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -34,6 +29,20 @@ pods:
           path: journal-data
           size: {{JOURNAL_DISK}}
           type: {{JOURNAL_DISK_TYPE}}
+    tasks:
+      bootstrap:
+        goal: FINISHED
+        cmd: >
+          mkdir -p journal-data/hdfs/current &&
```

Comment:
Can you please wrap this with echoes so it's easy to tell from logs if any given piece failed?

Also, echo the content of the api response.

e.g. 

`echo "creating version directory" && ... && echo "fetching version file" && ... && echo "version file has content $CONTENT" && ...`

Reply:
If you want to do this as a follow on PR, that's a-ok with me (since that will just run HDFS)

Reply:
Done